data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -456,11 +458,13 @@
                                                 [/row]
                                                 [row]
                                                     [column]
+                                                    	  # wmllint: markcheck off
```

Comment:
Again, making this translatable seems like the correct choice, rather than ignoring it.